From 02e6fef9253091c9f3963703c8ce017869177736 Mon Sep 17 00:00:00 2001
From: Yuxin Hu <yuxinhu@google.com>
Date: Mon, 30 Mar 2026 19:46:29 -0700
Subject: [PATCH 9/9] Update enable ANGLE warning message

The warning message is updated to align with partner's
internal style, as requested in b/493778441.

Flag: EXEMPT BUGFIX
Bug: b/270994705
Bug: b/459639258
Test: make
Cherrypick-From: https://googleplex-android-review.googlesource.com/q/commit:1b8ac1ca0011bc559d7bb4d0233ec88837c58399
Merged-In: I7af8133177c11ec0a3e05fcb480a8ee0dd66c8be
Change-Id: I7af8133177c11ec0a3e05fcb480a8ee0dd66c8be
---
 res/values/strings.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/res/values/strings.xml b/res/values/strings.xml
index 279f3d5bd03..3a950e1b3e5 100644
--- a/res/values/strings.xml
+++ b/res/values/strings.xml
@@ -12298,9 +12298,9 @@ Data usage charges may apply.</string>
     <!-- Debugging developer settings: enable angle as system driver? [CHAR LIMIT=50] -->
     <string name="enable_angle_as_system_driver">Experimental: Enable ANGLE</string>
     <!-- Debugging developer settings: enable angle as system driver summary [CHAR LIMIT=NONE] -->
-    <string name="enable_angle_as_system_driver_summary">Warning: Enable ANGLE as default OpenGL ES driver. This feature is in experiment and may not be compatible with some camera and video apps. Please backup your device data before turning on the switch</string>
+    <string name="enable_angle_as_system_driver_summary">Warning: ANGLE will be set as the default OpenGL ES driver. This feature is experimental and may not be compatible with some camera or video apps. Back up any important data before turning this on.</string>
     <!--Dialog body text used to explain a reboot is required after changing ANGLE as system GLES driver setting-->
-    <string name="reboot_dialog_enable_angle_as_system_driver">A reboot is required to change the system OpenGL ES driver. Please back up your device data before rebooting. In case the reboot failed, you need a factory reset to restore the device</string>
+    <string name="reboot_dialog_enable_angle_as_system_driver">To change the system OpenGL ES driver, the device will restart. Back up any important data before restarting. If restarting fails, a factory reset will be needed to use the device.</string>
 
     <!-- Title for App Compatibility Changes dashboard where developers can configure per-app overrides for compatibility changes [CHAR LIMIT=50] -->
     <string name="platform_compat_dashboard_title">App Compatibility Changes</string>
-- 
2.53.0

